bin/oscapd-evaluate

```diff
@@ -307,10 +307,13 @@ def cli_scan(args, config):
             )
             json_data["Profile"] = args.profile
 
-            with io.open(os.path.join(
-                    full_output_dir, "std.xml"), "w",
-                    encoding="utf-8") as f:
+            arf_filepath = os.path.join(full_output_dir, "std.xml")
+            with io.open(arf_filepath, "w", encoding="utf-8") as f:
                 f.write(standard_scan_results)
+            fix_script = oscap_helpers.generate_fix_from_result(config, arf_filepath)
```

Comment:
Please let's make this an option,` --fix_type` with choices: "bash", "ansible", "puppet". Let's do a little bit of extra work to make a good reusable CLI.